contrib/python/examples/run_example.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+export PYTHONPATH=..
+
+function examples {
+  for file in $@; do
+    python3 -c "import ast; f=open('"${file}"'); t=ast.parse(f.read()); print(ast.get_docstring(t) + '  -- "${file}"')"
+  done
+}
+
+while getopts "lh" arg; do
+  case $arg in
+    l ) examples $(ls eg_*.py |grep -v $(basename $0)); exit 0 ;;
```

Comment:
Could this just be:

```sh
l ) examples eg_*.py; exit ;;
```

I don't see the point of either the `grep -v ...` filter (maybe it's a hold-over from an iteration where you didn't pass `eg_*.py` to `ls`?) or the forced exit code (if `examples` returns an error, I expect the caller will want to know about it).